activerecord/test/cases/helper.rb

```diff
@@ -185,3 +185,7 @@ def in_time_zone(zone)
 end
 
 require "mocha/setup" # FIXME: stop using mocha
+
+def frozen_error_class
```

Comment:
Let's define it in `ActiveRecord::TestCase`.
https://github.com/rails/rails/blob/master/activerecord/test/cases/test_case.rb

Reply:
Thanks for the review. Updated.